iothub/device/src/Http1TransportSettings.cs

```diff
@@ -15,20 +15,36 @@ public sealed class Http1TransportSettings : ITransportSettings
     {
         private static readonly TimeSpan s_defaultOperationTimeout = TimeSpan.FromSeconds(60);
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Http1TransportSettings"/> class with default proxy settings.
```

Comment:
Does "Default proxy settings" mean that no proxy will be used?

Reply:
default proxy behavior - whatever is specified at your OS level